.github/workflows/ci-main.yml

```diff
@@ -194,9 +183,6 @@ jobs:
             if [[ '${{ needs.config.outputs.skip_long_tests }}' == 'true' ]]; then
               targets_args+=("--skip_long_tests")
             fi
-            if [[ '${{ needs.config.outputs.skip_didc_checks }}' == 'true' ]]; then
-              targets_args+=("--exclude_tags=didc")
```

Comment:
Don't we need to always exclude this tag now that there's a dedicated job that takes `CI_OVERRIDE_DIDC_CHECK` into account?

Reply:
> It will test the commit against itself

This is what I didn't realize: I suppose it tests against itself because you don't set `DID_CHECK_REV` in this job, right?

Reply:
That is correct! This is not new behavior however, this is already the case in `master`.